api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -246,6 +251,16 @@ private int logCompletionChange(
     return logChange(user, oldTime, newTime, accessModuleLogText.get(module) + " completion");
   }
 
+  private List<String> getAccessTiers(DbUser user) {
+    return accessTierService.getAccessTiersForUser(user).stream()
+        .map(DbAccessTier::getShortName)
+        .collect(Collectors.toList());
+  }
+
+  private int logAccessTierChange(DbUser user, List<String> oldTiers, List<String> newTiers) {
+    return logChange(user, oldTiers, newTiers, "Data access tiers");
```

Comment:
tested locally with a local API+UI and enabling/disabling Google 2FA for my local user.